app/assets/javascripts/darkswarm/directives/country_selector.js.erb.coffee

```diff
@@ -1,8 +1,8 @@
-Darkswarm.directive "countrySelector", () ->
+Darkswarm.directive "countrySelector", (RegistrationService) ->
   restrict: "A"
   require: 'ngModel'
   link: (scope, elem, attrs)->
-    defaultCountry = elem[0].getAttribute('data-default')
```

Comment:
It seems more transparent and less coupled to Spree to pass in a default value via the "data-default" attribute as seems to be the previous behaviour. Is there any reason we're changing to injecting the default directly into the JS rather than passing it as an HTML attribute to the directive?